Makefile

```diff
@@ -38,3 +38,4 @@ test-assets:
 	python pytestgeventwrapper.py rotkehlchen/tests/exchanges/test_independentreserve.py::test_assets_are_known
 	python pytestgeventwrapper.py rotkehlchen/tests/unit/test_zerionsdk.py::test_protocol_names_are_known
 	python pytestgeventwrapper.py rotkehlchen/tests/unit/test_zerionsdk.py::test_query_all_protocol_balances_for_account
+	python pytestgeventwrapper.py rotkehlchen/tests/data_import/
```

Comment:
I don't think that this change needed, since as you can see from the command name `test-assets`, it is supposed to test assets, not data import